modules/imgcodecs/src/grfmt_jpeg2000_openjpeg.hpp

```diff
@@ -61,13 +61,12 @@ using ImagePtr = std::unique_ptr<opj_image_t, detail::OpjImageDeleter>;
 
 } // namespace detail
 
-class Jpeg2KOpjDecoder CV_FINAL : public BaseImageDecoder
+class Jpeg2KOpjDecoderBase : public BaseImageDecoder
 {
 public:
-    Jpeg2KOpjDecoder();
-     ~Jpeg2KOpjDecoder() CV_OVERRIDE = default;
+    Jpeg2KOpjDecoderBase(OPJ_CODEC_FORMAT format);
+     ~Jpeg2KOpjDecoderBase() CV_OVERRIDE = default;
```

Comment:
You can safely remove `default` override of destructor (same for derived classes).

Reply:
👍